data-service/js/app.js

```diff
@@ -196,10 +196,13 @@ jQuery(function( $ ) {
 
     function getItems()
     {
-        try
-        {
-            return Backendless.Persistence.of( Task ).find().data;
-        }
+		try {
+			var entity = new Task(),
+				db = Backendless.Persistence.of( Task );
+			db.save(entity);
```

Comment:
I do not think it is necessary to store an object first. If table is not there, there will be an error which we can catch